content/references/protocol-reference/transactions/transaction-types/xchainaddattestation.md

```diff
@@ -0,0 +1,60 @@
+---
+html: xchainaddattestation.html 
+parent: transaction-types.html
+blurb: Create a bridge between two chains.
+labels:
+  - Interoperability
+status: not_enabled
+---
+# XChainAddAttestation
+
+The `XChainAddAddestation` transaction is submitted on the destination chain, by the witness server or anyone with access to the signatures from the witness server. It is a proof that an event (essentially just a locking/burning of funds) happened on the source chain.
+
+When enough witnesses have submitted their proofs on the destination chain that an event has occurred, the funds will be released to the destination account in the XChainCommit transaction, if specified. Otherwise, the claim ID owner must submit an XChainClaim transaction to determine where the funds will go on the destination chain.
+
+## Example {{currentpage.name}} JSON
+
+
+```json
+{
+  "TransactionType": "XChainAddAttestation",
```

Comment:
Are all of these transactions updated? This looks like an `XChainCreateBridge`, not an `XChainAddAttestation` (same with all the other transactions here)